Session/Conversations/ConversationVC.swift

```diff
@@ -334,12 +334,10 @@ final class ConversationVC : BaseVC, ConversationViewModelDelegate, OWSConversat
                 case .insert:
                     // Perform inserts before updates
                     self.messagesTableView.insertRows(at: [ IndexPath(row: Int(update.newIndex), section: 0) ], with: .fade)
-                    let viewItem = update.viewItem
-                    if viewItem?.interaction is TSOutgoingMessage {
-                        shouldScrollToBottom = true
-                    }
+                    shouldScrollToBottom = (self.scrollButton.alpha == 0)
```

Comment:
Rather than basing this on the alpha of the scroll button, let's based it on the table view's content offset.

Reply:
Both work, but using the content offset makes more sense semantically :)